Action/SetPayUAction.php

```diff
@@ -28,8 +28,8 @@ class SetPayUAction implements ApiAwareInterface, ActionInterface, GenericTokenF
      */
     protected $tokenFactory;
 
-    protected $api = array();
-
+    protected $api = array();   
+   
```

Comment:
There are a lot of unnecessary whitespaces. Please run the code through php-cs-fixer